ebusd-2.1.x/en/vaillant/08.hmu00.HW5103.SW0901.csv

```diff
@@ -0,0 +1,195 @@
+r,,EnergyConsumptionThisYear3,,,,,020300,month,,D1B,,kWh,,,,IGN:1,,,,,,energy,,,,,,,,,
+r,,EnergyConsumptionThisYear4,,,,,020400,month,,D1B,,kWh,,,,IGN:1,,,,,,energy,,,,,,,,,
+r,,EnergyConsumptionThisYear5,,,,,020500,month,,D1B,,kWh,,,,IGN:1,,,,,,energy,,,,,,,,,
+r,,EnergyConsumptionThisYear6,,,,,020600,month,,D1B,,kWh,,,,IGN:1,,,,,,energy,,,,,,,,,
+r,,EnergyConsumptionThisYear7,,,,,020700,month,,D1B,,kWh,,,,IGN:1,,,,,,energy,,,,,,,,,
+r,,EnergyConsumptionThisYear8,,,,,020800,month,,D1B,,kWh,,,,IGN:1,,,,,,energy,,,,,,,,,
+r,,EnergyConsumptionThisYear9,,,,,020900,month,,D1B,,kWh,,,,IGN:1,,,,,,energy,,,,,,,,,
+r,,EnergyConsumptionThisYear10,,,,,020A00,month,,D1B,,kWh,,,,IGN:1,,,,,,energy,,,,,,,,,
+r,,EnergyConsumptionThisYear11,,,,,020B00,month,,D1B,,kWh,,,,IGN:1,,,,,,energy,,,,,,,,,
+r,,EnergyConsumptionThisYear12,,,,,020C00,month,,D1B,,kWh,,,,IGN:1,,,,,,energy,,,,,,,,,
+
+*r,,,,,,B51A,05,,,IGN:3,,,,,,,,,,,,,,,,,,,,,
+*w,,,,,,B51A,06,,,,,,,,,,,,,,,,,,,,,,,,
+r,,YieldHeatingDay,,,,,FF3200,,,energy,10,kWh,,,,,,,,,,,,,,,,,,,
+r,,YieldCoolingDay,,,,,FF3201,,,energy,10,kWh,,,,,,,,,,,,,,,,,,,
+r,,YieldWarmWaterDay,,,,,FF3202,,,energy,10,kWh,,,,,,,,,,,,,,,,,,,
+r,,ch_3,,,,,FF3203,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,ch_4,,,,,FF3204,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,ch_5,,,,,FF3205,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,ch_6,,,,,FF3206,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,ch_7,,,,,FF3207,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r;w,,TargetTempWarmWater,(with intergrated System thermostat),,,,C63207,,,D2C,,°C,Target WarmWater temperature (°C),,,,,,,,,,,,,,,,,,
+r,,YieldHeatingMonth,,,,,FF320E,,,energy,10,kWh,,,,,,,,,,,,,,,,,,,
+r,,CopHeatingMonthSensor,,,,,FF320F,,,UIN,10, ,,,,,,,,,,,,,,,,,,,
+r,,YieldHeating,,,,,FF3210,,,energy,,kWh,Total Yield Heating Circuit,,,,,,,,,,,,,,,,,,
+r,,CopHeatingSensor,,,,,FF3211,,,UIN,10, ,Total Cop Heating Circuit,,,,,,,,,,,,,,,,,,
+r,,YieldWarmWaterMonth,,,,,FF3212,,,energy,10,kWh,,,,,,,,,,,,,,,,,,,
+r,,CopWarmWaterMonthSensor,,,,,FF3213,,,UIN,10, ,,,,,,,,,,,,,,,,,,,
+r,,YieldWarmWater,,,,,FF3216,,,energy,,kWh,Total Yield Hot Water Circuit,,,,,,,,,,,,,,,,,,
+r,,CopWarmWaterSensor,,,,,FF3217,,,UIN,10, ,Total Cop Hot Water Circuit,,,,,,,,,,,,,,,,,,
+r,,ch_18,,,,,FF3218,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r;w,,TargetTempHeating,(with intergrated System thermostat),,,,FF321C,,,D2C,,°C,Target temperature Heating (°C),,,,,,,,,,,,,,,,,,
+r,,CompressorBlocktimeStatus,,,,,FF321E,,,UIN,,min,,,,,,,,,,,,,,,,,,,
+r,,SupplyTempWeighted,,,,,FF321F,,,D2C,10,°C,,,,,,,,,,,,,,,,,,,
+r,,FlowTemp,,,,,FF3220,,,D2C,,°C,,,,,,,,,,,,,,,,,,,
+r,,EnergyIntegral,,,,,FF3221,,,integral,,°min,current energy integral in °min,,,,,,,,,,,,,,,,,,
+r,,SourceTempInput,,,,,FF3222,,,D2C,,°C,,,,,,,,,,,,,,,,,,,
+r,,CurrentYieldPower,,,,,FF3223,,,D1B,10,kW,,,,,,,,,,,,,,,,,,,
+r,,CurrentConsumedPower,,,,,FF3224,,,D1B,10,kW,,,,,,,,,,,,,,,,,,,
+r,,CompressorSensor,,,,,FF3225,,,percent3,10,,(percent of maximum),,,,,,,,,,,,,,,,,,
```

Comment:
FYI, for HMU00;0607;5103, I found UIN,16 works better than UIN,10, in case same applies here.